### PR TITLE
A collection of new IAM CLI examples

### PR DESCRIPTION
**Pull request by bisdavid**
*Description of changes:*
New examples for:
awscli/examples/iam/create-service-linked-role.rst
awscli/examples/iam/create-service-specific-credential.rst
awscli/examples/iam/delete-service-linked-role.rst
awscli/examples/iam/delete-service-specific-credential.rst
awscli/examples/iam/delete-ssh-public-key.rst
awscli/examples/iam/get-service-linked-role-deletion-status.rst
awscli/examples/iam/get-ssh-public-key.rst
awscli/examples/iam/list-policies-granting-service-access.rst
awscli/examples/iam/list-role-tags.rst
awscli/examples/iam/list-service-specific-credential.rst
awscli/examples/iam/list-ssh-public-keys.rst
awscli/examples/iam/list-user-tags.rst
awscli/examples/iam/reset-service-specific-credential.rst
awscli/examples/iam/tag-role.rst
awscli/examples/iam/tag-user.rst
awscli/examples/iam/untag-role.rst
awscli/examples/iam/untag-user.rst
awscli/examples/iam/update-role-description.rst
awscli/examples/iam/update-role.rst
awscli/examples/iam/update-service-specific-credential.rst
awscli/examples/iam/update-ssh-public-key.rst
awscli/examples/iam/upload-ssh-public-key.rst
awscli/examples/iam/wait/instance-profile-exists.rst
awscli/examples/iam/wait/policy-exists.rst
awscli/examples/iam/wait/role-exists.rst
awscli/examples/iam/wait/user-exists.rst

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/iam/list-ssh-public-keys.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  {
      "SSHPublicKeys": [
          {
              "UserName": "sofia",
              "SSHPublicKeyId": "APKA1234567890EXAMPLE",
              "Status": "Inactive",
              "UploadDate": "2019-04-18T17:04:49+00:00"
          }
      ]
  }
```
Comment (stealthycoin) [inline awscli/examples/iam/list-ssh-public-keys.rst]: Have some weird characters here can we get these replaced.

File: awscli/examples/iam/upload-server-certificate.rst [hunk 2]
```diff
@@ -1,15 +1,21 @@
 **To upload a server certificate to your AWS account**
```
Comment (stealthycoin) [inline awscli/examples/iam/upload-server-certificate.rst]: This file got made executable, can you set the permissions back to 644.

File: awscli/examples/iam/list-server-certificates.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+**To list the server certificates in your AWS account**
+
+The following ``list-server-certificates`` command lists all of the server certificates stored and available for use in your AWS account. ::
+
+    aws iam list-server-certificates
+
+Output:
+
+    {
```
Comment (stealthycoin) [inline awscli/examples/iam/list-server-certificates.rst]: Does this render correctly? The `rst` highlighter is not coloring it correctly here and the spacing is different than the other example files.

Comment (bisdavid) [reply awscli/examples/iam/list-server-certificates.rst]: I just fixed all of these.  As to the question about the last, yes, it renders correctly in my local build, which you can view at http://bisdavid.aka.amazon.com/docs/localcli/reference/iam/list-server-certificates.html    

I also added a few more (server-certificate CRUD commands) examples.

Please let me know if you find any other issues.

**Pull request merged**